core/modules/layout/tests/layout.test

```diff
@@ -42,13 +55,133 @@ class LayoutInterfaceTest extends BackdropWebTestCase {
     $this->test_node1 = $this->backdropCreateNode(array(
       'type' => 'post',
       'title' => $this->randomString(),
+      'uid' => $this->admin_user->uid,
     ));
+
     $this->test_node2 = $this->backdropCreateNode(array(
       'type' => 'page',
       'title' => $this->randomString(),
+      'uid' => $this->web_user->uid,
+    ));
+  }
+
+  /**
+   * Creates a new file field.
+   */
+  function addFileField($node) {
```

Comment:
We probably no longer need this method, since we're only testing node author relationships now.